config/kubernetes/README.md

```diff
@@ -0,0 +1,59 @@
+# Deploy Key Broker Service on Kubernetes
+
+We will see how to deploy KBS (with builtin Attestation Service) on a Kubernetes cluster.
+
+## The secrets
+
+Create a secret that you want to be served using this instance of KBS:
+
+```bash
+echo "This is my super secert" > overlays/key.bin
+```
+
+If you have more than one secrets, copy them over to the `config/kubernetes/overlays` directory and add those to the `overlays/kustomization.yaml` file after as shown below:
+
+```yaml
+...
+- name: keys
+  files:
+  - key.bin
+  - secret.key
+  - passowrd.txt
+  ...
+```
+
+## Defining KBS repositories
+
+With the default configuration the keys will be stored in `reponame/workload_key/`. If you wish to define a different repository make necessary changes to the `overlays/patch.yaml` file.
+
+## Deploy KBS
+
+Deploy KBS by running the following command:
+
+```bash
+./deploy-kbs.sh
```

Comment:
When following this, I currently get the following error:

```
  Warning  FailedMount  118s                   kubelet            Unable to attach or mount volumes: unmounted volumes=[keys], unattached volumes=[kube-api-access-ztkvh kbs-auth-public-key kbs-config as-config keys]:
 timed out waiting for the condition                                                                                                                                                                                    
  Warning  FailedMount  20s (x12 over 8m34s)   kubelet            MountVolume.SetUp failed for volume "keys" : secret "keys" not found    
```

Reply:
The namespace is missing in the overlay kustomize, see https://github.com/confidential-containers/kbs/pull/84#discussion_r1226498079